symplyphysics/laws/kinematic/vector/rolling_velocity_is_translational_plus_rotational_velocities.py

```diff
@@ -0,0 +1,40 @@
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    QuantityVector,
+    add_cartesian_vectors,
+)
+
+# Description
+## Suppose an object is rolling smoothly along a surface, i.e. without slipping or bouncing on the
+## surface. Such motion can be viewed as a combination of translational motion of the object's center
+## of mass, and rotational motion of its points about the rotational axis.
+
+# Law: v = v_translation + v_rotation
+## v - velocity vector of a point on the object.
+## v_translation - velocity component responsible for translational movement of object's center
+## v_rotation - velocity component responsible for rotational movement of the point on object,
+##              see [rotational velocity formula](./linear_velocity_is_angular_velocity_cross_radius.py)
+
+# Notes
+## Suppose the object in question is a wheel of constant radius, then
+## - v_translation is a vector related to the movement of the wheel's center
+##   its magnitude is given by [linear velocity formula](../linear_velocity_from_angular_velocity_and_radius)
+##   where radius is the total radius of the wheel
+## - v_rotation is calculated via [rotational velocity formula](./linear_velocity_is_angular_velocity_cross_radius.py)
```

Comment:
It is the same formula as `linear_velocity_from_angular_velocity_and_radius` but for vectors.